awpy/parser/parse_demo.go

```diff
@@ -1461,6 +1477,14 @@ func registerRoundEndHandler(demoParser *dem.Parser, currentGame *Game, currentR
 	})
 }
 
+func registerRoundMVPHandler(demoParser *dem.Parser, currentGame *Game, currentRound *GameRound) {
+	(*demoParser).RegisterEventHandler(func(e events.RoundMVPAnnouncement) {
+	    currentRound.MVPName = e.Player.Name
+	    currentRound.MVPSteamID = int64(e.Player.SteamID64)
+	    currentRound.MVPReason = convertRoundMVPReason(e.Reason)
+	})
```

Comment:
I think to be clean we need a nil check here for `e.Player`?

Reply:
Done